Sources/IO/IOCaptureSession.swift

```diff
@@ -21,8 +20,10 @@ final class IOCaptureSession {
             return false
         }
     }
-    #else
+    #elseif os(macOS)
     static let isMultiCamSupported = true
+    #elseif os(visionOS)
+    static let isMultiCamSupported = false
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/shogo4405/HaishinKit.swift/blob/52fa1eff4cbfe2963a5275575069a2cc0ae108a0/Sources/IO/IOCaptureSession.swift#L26"></span>The 'stat' is the require reason api. Please check privacy manifest files.</td>
    </tr>
  </tbody>
</table>